src/graph.erl

```diff
@@ -43,175 +38,166 @@
     list_neighbours/1,
     list_parents/1,
     list_children/1
-  ]
-).
+]).
 
 -include("records.hrl").
 
 %%%---------------------------
 %% VERTICES API
 %%%---------------------------
 
-generate_id() ->
-  Id =
-    <<
-      <<Y>> || <<X : 4>> <= crypto:hash(md5, term_to_binary(make_ref())),
-      Y <- integer_to_list(X, 16)
-    >>,
-  Zone = list_to_binary(?ZONE_ID),
-  IdWithZone = <<Zone/binary, "_", Id/binary>>,
-  case persistence:exists(IdWithZone) of
-    {ok, false} -> {ok, IdWithZone};
-    {ok, true} -> generate_id();
-    {error, Reason} -> {error, Reason}
-  end.
+-spec generate_id(binary()) -> {ok, binary()} | {error, any()}.
+generate_id(Name) ->
+    Zone = list_to_binary(?ZONE_ID),
+    Id = gmm_utils:create_vertex_id(Name, Zone),
+%%    IdWithZone = <<Zone/binary, "_", Id/binary>>,
```

Comment:
Ta linijka kodu po prostu czeka na swoją kolej, czy to nie potrzebne? Jak nie potrzebne, to usuń.

Reply:
Ta linijka została jakby ktoś kto będzie analogiczną funkcję w gmm_utils implementował chciał sobie na to spojrzeć. Po prostu temat tworzenia ID z imienia i zony jeszcze nie jest zamknięty.

Reply:
No, już to usunąłem. Faktycznie nie jest potrzebna